src/tests/test_raft_repl_dev.cpp

```diff
@@ -269,7 +269,10 @@ class TestReplicatedDB : public homestore::ReplDevListener {
                    "[Replica={}] Save logical snapshot callback obj_id={} term={} idx={} is_last={} num_items={}",
```

Comment:
nit: Move log one line  after updating the snp_data->offset.

Reply:
this log want to print out what we receive from leader , so I think we should keep it as this